_posts/2019-08-29-dplyr.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'DPLYR Blocks'
+type: 'DPLYR'
+layout: nil
+---
+
+
+<img src="/guide/images/dplyr_filter.png" width="30%">
```

Comment:
This is a case where I think the URL has to be run through `relative_url` *or* the link has to be relative e.g. `src="../images/dplyr_filter.png"` in order to work everywhere.